### DISCUSSION TITLE
SMTP Authentication with the remote server failed

### DISCUSSION THREAD

**Discussion by abkiwi9** [postalserver/postal#3090] [Help with using Postal]
Hi,

I have been using Postal for a few years now with no problems, however, I updated to version 3.3.4 last month and have just noticed all of our SMTP connections from SmarterMail to Postal no longer work.  All I get when checking or setting up a new SMTP Gateway is:  Authentication with the remote server failed.

I have done all the checks and everything works and is correct.  I have tested setting a new SMTP gateway from SmarterMail to SMTP2GO and this works fine.  Our postal mail stopped working at the same time we did the update.   Email has still worked as we have the main server set as a second backup relay.

How would I fix this problem or is an update needed to fix this?

Thank you
Regards
Aaron

**Comment by willpower232** [comment]
If you try `postal log smtp` you should be able to see what your connections are trying with Postal to see what errors there are.

If there are no logs at all then perhaps there is a firewall upset or something isn't running in your `postal status`

**Comment by abkiwi9** [comment]
Thanks, I have checked the firewall, and it is turned off.  There are no logs, I have turned logging on, and check the postal status as follows:

NAME                COMMAND                  SERVICE             STATUS
postal-smtp-1       "/docker-entrypoint.…"   smtp                running
postal-web-1        "/docker-entrypoint.…"   web                 running
postal-worker-1     "/docker-entrypoint.…"   worker              running

No PORTS are showing.  It has been working fine up until the update.
